components/sample-data-service/server/server.go

```diff
@@ -0,0 +1,69 @@
+package server
+
+import (
+	"context"
+	"net"
+
+	grpc_middleware "github.com/grpc-ecosystem/go-grpc-middleware"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/reflection"
+	"google.golang.org/grpc/status"
+
+	"github.com/chef/automate/components/sample-data-service/service"
+	"github.com/chef/automate/lib/grpc/health"
+	"github.com/chef/automate/lib/tracing"
+)
+
+// Server is an sample-data server
+type Server struct {
+	service *service.Service
+}
+
+// NewServer returns an sample-data server
+func NewServer(service *service.Service) *Server {
+	return &Server{service: service}
+}
+
+// NewGRPCServer creates a grpc server that serves all Teams GRPC APIs
```

Comment:
Teams??

Reply:
copy-pasted stuff forgot to update comment thanks!